### ISSUE TITLE
If incorrect PIN is entered, then corrected, wallet does not load

### ISSUE THREAD

**Issue by bitcoin-solutions** [bitcoin-solutions/multibit-hd#336]
If the user enters an incorrect PIN, then corrects it, the wallet does not load.
The log has the error:
```
java.lang.IllegalStateException: Optional.get() cannot be called on an absent value
	at com.google.common.base.Absent.get(Absent.java:47)
	at org.multibit.hd.ui.views.wizards.credentials.CredentialsWizardModel.getOrCreateTrezorWallet(CredentialsWizardModel.java:767)
	at org.multibit.hd.ui.views.wizards.credentials.CredentialsWizardModel.access$400(CredentialsWizardModel.java:54)
```
I suspect the HardwareWalletContext is being reset on failure and the DeterministicKey is being deleted.

**Comment by bitcoin-solutions** [comment]
Fixed at same time as #335.
Awaiting review and closing

**Comment by gary-rowe** [comment]
Verified working.

Closing.
